automation_tools/__init__.py

```diff
@@ -444,6 +444,41 @@ def setup_default_libvirt(bridge=None, ip_address="192.168.100.1"):
     return interface
 
 
+def setup_http_proxy():
+    """Postinstall task to set http_proxy and content_default_http_proxy settings for
+    Satellite >= 6.7
+
+    Proxy information is passed using the PROXY_INFO environmental variable.
+    The expected format is::
+
+        PROXY_INFO=proxy://<username>:<password>@<hostname>:<port>
+
+    ``username`` and ``password`` fields can be omitted if the proxy does not
+    require authentication, for example::
+
+        PROXY_INFO=proxy://<hostname>:<port>
+    """
+    proxy_info = os.environ.get('PROXY_INFO')
+    command = (
+        'hammer -u admin -p {0} settings set --name "http_proxy" --value {1}'
+    ).format(os.environ.get('ADMIN_PASSWORD', 'changeme'), proxy_info.replace('proxy:', 'http:'))
+    run(command, warn_only=True)
+    proxy_info = urlsplit(proxy_info)
+    url = 'http://{0}:{1}'.format(proxy_info.hostname, proxy_info.port)
+    command = (
+        'hammer -u admin -p {0} http-proxy create --name {1} --url {2}'
+    ).format(os.environ.get('ADMIN_PASSWORD', 'changeme'), proxy_info.hostname, url)
+    if proxy_info.username and proxy_info.password is not None:
```

Comment:
Didn't you mean one of
```suggestion
    if proxy_info.username is not None and proxy_info.password is not None:
```
```suggestion
    if proxy_info.username and proxy_info.password:
```
It's little weird to explicitly check against None for one variable, but for falseness for another.

Reply:
Yes, Thanks. Updated PR